sw/device/tests/aes_force_prng_reseed_test.c

```diff
@@ -0,0 +1,264 @@
+  } else {
+    LOG_ERROR(
+        "ERROR:AES encryption did not stall as expected after EDN stop! Lets "
+        "proceed "
+        "for now to verify further steps...");
+  }
+
+  log_csrng_internal_state(&csrng);
+  monitor_entropy_sources();
+
+  // Expanding aes_testutils_setup_encryption T1 to T8:
+  // T: Start of aes_testutils_setup_encryption function steps
+  // T1: Mask the key. Note that this should not be done manually. Software is
+  // expected to get the key in two shares right from the beginning.
+  uint8_t key_share0[sizeof(kAesModesKey128)];
+  for (int i = 0; i < sizeof(kAesModesKey128); ++i) {
+    key_share0[i] = kAesModesKey128[i] ^ kKeyShare1[i];
+  }
+
+  // T2: "Convert" key share byte arrays to `dif_aes_key_share_t`.
+  dif_aes_key_share_t key;
+  memcpy(key.share0, key_share0, sizeof(key.share0));
+  memcpy(key.share1, kKeyShare1, sizeof(key.share1));
+
+  // T3:  Wait for status kDifAesStatusIdle
+  // Start AES with the given transaction
+  AES_TESTUTILS_WAIT_FOR_STATUS(&aes, kDifAesStatusIdle, true, kTestTimeout);
+
+  // T4: dif_aes_start
+  CHECK_DIF_OK(dif_aes_start(&aes, &transaction, &key, NULL));
+
+  LOG_INFO("Disabling CSRNG...");
+  CHECK_DIF_OK(dif_csrng_stop(&csrng));
+  CHECK_DIF_OK(dif_entropy_src_stop(&entropy_src));
+
+  // Re-enable Entropy Source and CSRNG in the proper sequence
+  LOG_INFO("Re-initializing Entropy Source and CSRNG...");
+
+  // Initialize and configure Entropy Source
+  dif_entropy_src_config_t entropy_src_config =
+      entropy_testutils_config_default();
+  CHECK_DIF_OK(dif_entropy_src_configure(&entropy_src, entropy_src_config,
+                                         kDifToggleEnabled));
+
+  // Initialize and start CSRNG
+  CHECK_DIF_OK(dif_csrng_configure(&csrng));
+  // set_edn_auto_mode();
+  LOG_INFO("Setting EDN to auto mode...");
+  TRY(entropy_testutils_auto_mode_init());
+  // CHECK_DIF_OK(entropy_testutils_auto_mode_init());
+  monitor_edn_status(&edn0, &edn1);
+
+  // Trigger PRNG reseed request
+  LOG_INFO("Triggering PRNG reseed...");
+  CHECK_DIF_OK(dif_aes_trigger(&aes, kDifAesTriggerPrngReseed));
+
+  // T5: "Convert" plain data byte arrays to `dif_aes_data_t`
+  dif_aes_data_t in_data_plain;
+  memcpy(in_data_plain.data, kAesModesPlainText, sizeof(in_data_plain.data));
+
+  // T6: Wait ofr Status kDifAesStatusIdle
+  LOG_INFO(
+      "Now Starting AES encryption process firstly by observing if AES is in "
+      "Idle state");
+  AES_TESTUTILS_WAIT_FOR_STATUS(&aes, kDifAesStatusIdle, true, kTestTimeout);
+
+  // T7: Wait for Status input DifAesStatusInputReady
+  LOG_INFO("AES started: ECB encryption, waiting for input ready.");
+  AES_TESTUTILS_WAIT_FOR_STATUS(&aes, kDifAesStatusInputReady, true,
+                                kTestTimeout);
+  // T8: Load the plain text to trigger the encryption operation
+  CHECK_DIF_OK(dif_aes_load_data(&aes, in_data_plain));
+
+  // Trigger AES reseed request, specific for manual operation case
```

Comment:
`kDifAesTriggerStart` is not the same as `kDifAesTriggerPrngReseed`.

Reply:
Yes. Can you clarify more on this review comment wrt code?